tests/Bootstrap.php

```diff
@@ -315,6 +315,13 @@ public function initialize(Container $container) {
             ->rule('HtmlFormatter')
             ->setClass(\VanillaHtmlFormatter::class)
             ->setShared(true)
+
+            ->rule(Vanilla\Scheduler\SchedulerInterface::class)
```

Comment:
Unless the dummy scheduler can execute jobs as soon as they are scheduled then it's probably not right for testing. I would like unit tests to run jobs synchronously so that we can get coverage on the code in the jobs too.

Reply:
I don't _think_ there's a scheduler that currently does this, but I'm looking into it. We might just need to add one as part of this PR.

Reply:
I was wondering if we can subclass the dummy and have its schedule method just call dispatchAll. Definitely need @egarciajulia's input here.